cg/models/deliverables/metric_deliverables.py

```diff
@@ -119,6 +122,29 @@ def set_sample_ids(cls, _, values: dict) -> set:
         return set(sample_ids)
 
 
+class ConditionMetricsDeliverables(BaseModel):
+    """Specification for a metric deliverables file with conditions sets."""
+
+    metrics_: List[MetricsBase] = Field(..., alias="metrics")
+
+    @validator("metrics_", always=True)
```

Comment:
`always=True` is also not needed here

```suggestion
    @validator("metrics")
```

Reply:
If this value is missing then the model will throw a `ValidationError` as `metrics` is a required field. That's what we want, right?

Validate always is useful when you want to set dynamically a value during the instantiation of the model. And you usually use it with `pre=True`, as you want to assign the default values before the actual validation. 
It is also useful when you want it to be included in the output even if its value is None.

Reply:
ohh! I see.  Then you are right, I'll remove it. Thanks 💟